python3.7/Dockerfile

```diff
@@ -42,8 +43,11 @@ RUN curl -sL https://rpm.nodesource.com/setup_14.x | bash - && \
     yum install -y nodejs
 
 # Download and install Python 3.7
+# Need to remove symlink from /usr/bin/python3 -> /usr/bin/python3.6
+# so alinstall does not fail
 RUN curl https://www.python.org/ftp/python/3.7.0/Python-3.7.0.tgz -O -J -L && \
     tar xzf Python-3.7.0.tgz -C /usr/src/ && \
+    rm -f /usr/bin/python3 && \
```

Comment:
This is needed because for some reason python3 is already pointing (symlink) to python3.6 and altinstall is not able to overwrite.

See: https://github.com/inveniosoftware/docker-invenio/runs/2475233128?check_suite_focus=true#step:4:2919